pkg/registry/core/persistentvolumeclaim/strategy.go

```diff
@@ -122,8 +122,7 @@ func (persistentvolumeclaimStrategy) ValidateUpdate(ctx context.Context, obj, ol
 	newPvc := obj.(*api.PersistentVolumeClaim)
 	oldPvc := old.(*api.PersistentVolumeClaim)
 	opts := validation.ValidationOptionsForPersistentVolumeClaim(newPvc, oldPvc)
-	errorList := validation.ValidatePersistentVolumeClaim(newPvc, opts)
-	return append(errorList, validation.ValidatePersistentVolumeClaimUpdate(newPvc, oldPvc, opts)...)
+	return validation.ValidatePersistentVolumeClaimUpdate(newPvc, oldPvc, opts)
```

Comment:
Verified that ValidatePersistentVolumeClaimUpdate calls ValidatePersistentVolumeClaim internally